python/dgl/graphbolt/sampling_graph.py

```diff
@@ -0,0 +1,30 @@
+"""Sampling Graphs."""
+
+
+class SamplingGraph:
+    r"""Class for sampling graph."""
+
+    def __init__(self):
+        pass
+
+    @property
+    def num_nodes(self) -> int:
```

Comment:
Is there a need to update this to ensure consistency with `CSCSamplingGraph.num_nodes()`?